lib/provider.js

```diff
@@ -205,8 +205,14 @@ export default function() {
                 return;
             }
             unregisterToStateChangeStartEvent();
-            var statePromise = $state.go(state, stateParams, stateOptions);
             transition(transitionOptions);
+            if (ionic.Platform.isIOS()) {
```

Comment:
might be useful to have the timeout also for Android. I had transition issues from time to time on Android and it seems like timeout makes it better.